src/gluonnlp/model/transformer.py

```diff
@@ -111,7 +118,8 @@ def __init__(self, units=512, hidden_size=2048, dropout=0.0, use_residual=True,
                                   prefix='ffn_2_')
             if dropout:
                 self.dropout_layer = nn.Dropout(rate=dropout)
-            self.layer_norm = _get_layer_norm(use_bert_layer_norm, units)
+            self.layer_norm = _get_layer_norm(use_bert_layer_norm, units,
+                                              layer_norm_eps=layer_norm_eps)
 
     def _get_activation(self, act):
         """Get activation block based on the name. """
```

Comment:
what if act is None?

Reply:
seems None will not be passed to self._get_activation()
`self.activation = self._get_activation(activation) if activation else None `